### ISSUE TITLE
Skip `bnd-maven-plugin` configuration when `bnd.skip` is present

### ISSUE THREAD

**Issue by vy** [apache/logging-parent#34]
When `bnd.skip` is present, the `bnd-maven-plugin` configuration still kicks in and causes a

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:3.1.1:install (default-install) on project log4j-transform-maven-shade-plugin-extensions: The packaging for this project did not assign a file to the build artifact
```

failure.

**Comment by ppkarwasz** [comment]
This is probably caused by the `<extensions>true</extensions` setting, which should be changed to `false` in such a case.

**Comment by vy** [comment]
Shall we put the `bnd-maven-plugin` under `project > build > plugins` to a separate profile activated via `bnd.skip` instead? Or does this sound more like a bug that should be filed for `bnd-maven-plugin`?

**Comment by ppkarwasz** [comment]
Sounds more like a bug to me.

I can try to fix it or report it.

**Comment by vy** [comment]
Indirectly fixed by #69.
